CosmosToPostgreSQL/CosmosToPostgreSQL/Program.cs

```diff
@@ -429,14 +453,64 @@ await using (NpgsqlDataReader reader = await pgcom.ExecuteReaderAsync())
                 }
             }
 
-            if (internalId == 0)
+            if (internalId == 0 && !_dataelementWhitelist.Contains(element.InstanceGuid))
```

Comment:
It is going to be confusing that `_dataelementWhitelist` is a list of instanceGuids.

Reply:
The instance guid is the key for the instances that are white listed for the data elements